### ISSUE TITLE
Setting self.window.isHidden on iOS causes app to crash

### ISSUE THREAD

**Issue by digibum** [flutter/flutter#59137]
## The goal
is to hide app content from user when in app switcher. We achieved this by overriding two methods (applicationWillResignActive, applicationDidBecomeActive) in AppDelegate.swift file. See code below for implementation.

## The issue
when user minimises app, then reopens it and then clicks any TextFormField the app will crash. This issue is also described here https://github.com/flutter/flutter/issues/26028

## Steps to Reproduce
main.dart
```dart
import 'package:flutter/material.dart';

void main() => runApp(MyApp());

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      home: Scaffold(
        body: Center(
          child: TextFormField(),
        ),
      ),
    );
  }
}
```
AppDelegate.swift

```swift
import UIKit
import Flutter

@UIApplicationMain
@objc class AppDelegate: FlutterAppDelegate {
  override func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
    GeneratedPluginRegistrant.register(with: self)
    return super.application(application, didFinishLaunchingWithOptions: launchOptions)
  }

  override func applicationWillResignActive(_ application: UIApplication) {
    self.window.isHidden = true;
  }
  override func applicationDidBecomeActive(_ application: UIApplication) {
    self.window.isHidden = false;
  }
}
```

## Fluter doctor

<details>
<summary>flutter doctor -v</summary>

```bash
> [√] Flutter (Channel stable, v1.17.2, on Microsoft Windows [Version 10.0.18363.657], locale en-SI)
>     • Flutter version 1.17.2 at F:\Flutter
>     • Framework revision 5f21edf8b6 (12 days ago), 2020-05-28 12:44:12 -0700
>     • Engine revision b851c71829
>     • Dart version 2.8.3
> 
> [!] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
>     • Android SDK at F:\Android\Sdk
>     • Platform android-29, build-tools 29.0.2
>     • ANDROID_HOME = F:\Android\Sdk
>     • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
>     • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b03)
>     X Android license status unknown.
>       Try re-installing or updating your Android SDK Manager.
>       See https://developer.android.com/studio/#downloads or visit visit https://flutter.dev/docs/get-started/install/windows#android-setup for detailed instructions.
> 
> [√] Android Studio (version 3.5)
>     • Android Studio at C:\Program Files\Android\Android Studio
>     • Flutter plugin version 42.1.1
>     • Dart plugin version 191.8593
>     • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b03)
> 
> [√] VS Code (version 1.45.1)
>     • VS Code at C:\Users\Matej\AppData\Local\Programs\Microsoft VS Code
>     • Flutter extension version 3.11.0
> 
> [!] Connected device
>     ! No devices available
```
</details>

**Comment by TahaTesser** [comment]
Issue is reproducible 

<details> 
<summary>flutter doctor -v</summary> 

``` 
[✓] Flutter (Channel dev, 1.19.0-5.0.pre, on Mac OS X 10.15.5 19F101, locale
    en-GB)
    • Flutter version 1.19.0-5.0.pre at /Users/tahatesser/Code/flutter_dev
    • Framework revision c264b70ec3 (2 days ago), 2020-06-08 09:27:34 -0700
    • Engine revision c7f9725521
    • Dart version 2.9.0 (build 2.9.0-13.0.dev 87c5e3612b)

 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
    • Android SDK at /Users/tahatesser/Code/sdk
    • Platform android-29, build-tools 29.0.3
    • ANDROID_HOME = /Users/tahatesser/Code/sdk
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build
      1.8.0_242-release-1644-b3-6222593)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.5)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.5, Build version 11E608c
    • CocoaPods version 1.9.3

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.0)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 46.0.2
    • Dart plugin version 193.7361
    • Java version OpenJDK Runtime Environment (build
      1.8.0_242-release-1644-b3-6222593)

[✓] VS Code (version 1.45.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.11.0

[✓] Connected device (5 available)
    • SM M305F      • 32003c30dc19668f          • android-arm64  • Android 10
      (API 29)
    • Taha’s iPhone • 00008020-001059882212002E • ios            • iOS 13.5.1
    • macOS         • macOS                     • darwin-x64     • Mac OS X
      10.15.5 19F101
    • Web Server    • web-server                • web-javascript • Flutter Tools
    • Chrome        • chrome                    • web-javascript • Google Chrome
      83.0.4103.97

• No issues found!
``` 

</details>

**Comment by vanlooverenkoen** [comment]
This is the stack trace of the crash I am having. Similar as https://github.com/flutter/flutter/issues/26028
```
*** First throw call stack:
(
	0   CoreFoundation                      0x00007fff23e3de6e __exceptionPreprocess + 350
	1   libobjc.A.dylib                     0x00007fff512539b2 objc_exception_throw + 48
	2   CoreFoundation                      0x00007fff23e3dbe8 +[NSException raise:format:arguments:] + 88
	3   Foundation                          0x00007fff258d6bd2 -[NSAssertionHandler handleFailureInMethod:object:file:lineNumber:description:] + 191
	4   Flutter                             0x000000010ff9bb0f -[FlutterTextInputPlugin showTextInput] + 298
	5   Flutter                             0x000000010ff9b8fd -[FlutterTextInputPlugin handleMethodCall:result:] + 149
	6   Flutter                             0x0000000110019b1f __45-[FlutterMethodChannel setMethodCallHandler:]_block_invoke + 104
	7  <…>
Lost connection to device.
```

**Comment by HannesVDB** [comment]
I would suggest to do something like this.

`
import UIKit
import Flutter

@UIApplicationMain
@objc class AppDelegate: FlutterAppDelegate {
    private var containerView: UIView?
    
    override func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
        GeneratedPluginRegistrant.register(with: self)
        return super.application(application, didFinishLaunchingWithOptions: launchOptions)
    }
    
    
    override func applicationWillResignActive(_ application: UIApplication) {
        containerView = UIView(frame: window.bounds)
        guard let view = containerView else { return }
        view.backgroundColor = UIColor.black
        window.addSubview(view)
    }
    
    override func applicationDidBecomeActive(_ application: UIApplication) {
        guard let view = containerView else {return }
        view.removeFromSuperview()
    }
}
`

**Comment by vanlooverenkoen** [comment]
**This works but is a workaround.** 

` self.window.isHidden = false;` is something that should be fixed by the Flutter team

**Comment by dreampowder** [comment]
addition to @HannesVDB 's answer, i improved the setting a little bit more ux friendly imho. 

`import UIKit
import Flutter

@UIApplicationMain
@objc class AppDelegate: FlutterAppDelegate {
    
    let TAG_BLUR_VIEW = 12345
    var blurView:UIVisualEffectView?
    
  override func application(
    _ application: UIApplication,
    didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?
  ) -> Bool {
    UIApplication.shared.isIdleTimerDisabled = true
    GeneratedPluginRegistrant.register(with: self)
    return super.application(application, didFinishLaunchingWithOptions: launchOptions)
  }
    
    override func applicationWillResignActive(_ application: UIApplication) {
        if let view = self.window.rootViewController?.view.subviews.first(where: {$0.tag == TAG_BLUR_VIEW}){
            view.removeFromSuperview()
            blurView = nil
        }
        if blurView == nil{
            blurView = UIVisualEffectView(frame: UIScreen.main.bounds)
            blurView?.effect = UIBlurEffect(style: .light)
            blurView?.tag = TAG_BLUR_VIEW
        }
        self.window.rootViewController?.view.insertSubview(blurView!, at: 0)
    }
    
    override func applicationDidBecomeActive(_ application: UIApplication) {
        if let view = self.window.rootViewController?.view.subviews.first(where: {$0.tag == TAG_BLUR_VIEW}){
            view.removeFromSuperview()
            blurView = nil
        }
    }
}
`

**Comment by edsonboldrini** [comment]
Is there any way to prevent screenshots on iOS, searching for this I ended here, and seems nothing works, very frustrating...

**Comment by TahaTesser** [comment]
<details>
<summary>Xcode logs</summary>

```bash
2021-01-19 19:31:03.253241+0530 Runner[8166:3997808] Metal API Validation Enabled
2021-01-19 19:31:03.465731+0530 Runner[8166:3997979] flutter: Observatory listening on http://127.0.0.1:62332/JMILMS8d-5o=/
2021-01-19 19:31:37.318315+0530 Runner[8166:3997808] *** Assertion failure in -[FlutterTextInputPlugin textInputParentView], FlutterTextInputPlugin.mm:1363
2021-01-19 19:31:37.320921+0530 Runner[8166:3997808] *** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'The application must have a key window since the keyboard client must be part of the responder chain to function'
*** First throw call stack:
(0x185cd186c 0x19ac40c50 0x185bd7000 0x186f6b91c 0x1053364cc 0x1053369d4 0x105335cdc 0x105335934 0x105bae95c 0x10534aba8 0x10564b7b0 0x1055ea0e4 0x1055ec79c 0x185c4dfa0 0x185c4dba0 0x185c4cffc 0x185c46ee4 0x185c4621c 0x19d74a784 0x188684fe0 0x18868a854 0x104e25a90 0x1859066b0)
libc++abi.dylib: terminating with uncaught exception of type NSException
*** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'The application must have a key window since the keyboard client must be part of the responder chain to function'
terminating with uncaught exception of type NSException
(lldb) 
```
</details>


<details>
<summary>logs</summary>

```bash
[        ] DevFS: Sync finished
[        ] Syncing files to device Taha’s iPhone... (completed in 175ms)
[        ] Synced 0.0MB.
[        ] <- accept
[  +17 ms] Connected to _flutterView/0x106825020.
[   +3 ms] Flutter run key commands.
[   +2 ms] r Hot reload. 🔥🔥🔥
[   +1 ms] R Hot restart.
[        ] h Repeat this help message.
[        ] d Detach (terminate "flutter run" but leave application running).
[        ] c Clear the screen
[        ] q Quit (terminate the application on the device).
[        ] An Observatory debugger and profiler on Taha’s iPhone is available at: http://127.0.0.1:53128/x-Kq7c6TtLo=/
[   +1 ms] 
           Flutter DevTools, a Flutter debugger and profiler, on Taha’s iPhone is available at: http://127.0.0.1:9100?uri=http%3A%2F%2F127.0.0.1%3A53128%2Fx-Kq7c6TtLo%3D%2F
[        ] Running with unsound null safety
[        ] For more information see https://dart.dev/null-safety/unsound-null-safety
[+15517 ms] *** Assertion failure in -[FlutterTextInputPlugin textInputParentView], FlutterTextInputPlugin.mm:1363
[   +1 ms] *** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'The application must have a key window since the keyboard client must be part of the responder chain to function'
[        ] *** First throw call stack:
[        ] (0x185cd186c 0x19ac40c50 0x185bd7000 0x186f6b91c 0x100c164cc 0x100c169d4 0x100c15cdc 0x100c15934 0x10148e95c 0x100c2aba8 0x100f2b7b0 0x100eca0e4 0x100ecc79c 0x185c4dfa0 0x185c4dba0 0x185c4cffc 0x185c46ee4 0x185c4621c 0x19d74a784 0x188684fe0 0x18868a854 0x100895a90 0x1859066b0)
[        ] libc++abi.dylib: terminating with uncaught exception of type NSException
[ +153 ms] Process 8159 stopped
[        ] * thread #1, queue = 'com.apple.main-thread', stop reason = signal SIGABRT
[        ]     frame #0: 0x00000001b3a48414 libsystem_kernel.dylib`__pthread_kill + 8
[        ] libsystem_kernel.dylib`__pthread_kill:
[        ] ->  0x1b3a48414 <+8>:  b.lo   0x1b3a48434               ; <+40>
[        ]     0x1b3a48418 <+12>: pacibsp 
[        ]     0x1b3a4841c <+16>: stp    x29, x30, [sp, #-0x10]!
[        ]     0x1b3a48420 <+20>: mov    x29, sp
[        ] Target 0: (Runner) stopped.
[   +1 ms] ios-deploy exited with code 0
[ +103 ms] Service protocol connection closed.
[        ] Lost connection to device.
[   +2 ms] DevFS: Deleting filesystem on the device (file:///private/var/mobile/Containers/Data/Application/023F96E5-D6EC-462B-921C-28E9E3E36FC1/tmp/master_flutteriC20LG/master_flutter/)
[ +258 ms] Ignored error while cleaning up DevFS: TimeoutException after 0:00:00.250000: Future not completed
[   +3 ms] "flutter run" took 76,686ms.
[ +162 ms] ensureAnalyticsSent: 156ms
[   +2 ms] Running shutdown hooks
[        ] Shutdown hook priority 4
[   +1 ms] Shutdown hooks complete
[        ] exiting with code 0
```
</details>
<details>
<summary>flutter doctor -v</summary>

```bash
[✓] Flutter (Channel stable, 1.22.5, on macOS 11.1 20C69 darwin-x64, locale en-GB)
    • Flutter version 1.22.5 at /Users/tahatesser/Code/flutter_stable
    • Framework revision 7891006299 (6 weeks ago), 2020-12-10 11:54:40 -0800
    • Engine revision ae90085a84
    • Dart version 2.10.4

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /Volumes/Extreme/SDK
    • Platform android-30, build-tools 30.0.3
    • ANDROID_HOME = /Volumes/Extreme/SDK
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 12.3)
    • Xcode at /Volumes/Extreme/Xcode.app/Contents/Developer
    • Xcode 12.3, Build version 12C33
    • CocoaPods version 1.10.1

[!] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)

[✓] VS Code (version 1.52.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.18.1

[✓] Connected device (1 available)
    • Taha’s iPad (mobile) • 00008020-000255113EE8402E • ios • iOS 14.3
    ! Device emulator-5554 is offline.

! Doctor found issues in 1 category.
```
```bash
[✓] Flutter (Channel master, 1.26.0-2.0.pre.402, on macOS 11.1 20C69 darwin-x64, locale en-GB)
    • Flutter version 1.26.0-2.0.pre.402 at /Users/tahatesser/Code/flutter_master
    • Framework revision 2a188eeca3 (33 hours ago), 2021-01-17 19:27:00 -0800
    • Engine revision 609036f2bf
    • Dart version 2.12.0 (build 2.12.0-236.0.dev)

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /Volumes/Extreme/SDK
    • Platform android-30, build-tools 30.0.3
    • ANDROID_HOME = /Volumes/Extreme/SDK
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Volumes/Extreme/Xcode.app/Contents/Developer
    • Xcode 12.3, Build version 12C33
    • CocoaPods version 1.10.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)

[✓] VS Code (version 1.52.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.18.1

[✓] Connected device (4 available)
    • Android SDK built for x86 (mobile) • emulator-5554             • android-x86    • Android 10 (API 29) (emulator)
    • Taha’s iPad (mobile)               • 00008020-000255113EE8402E • ios            • iOS 14.3
    • macOS (desktop)                    • macos                     • darwin-x64     • macOS 11.1 20C69 darwin-x64
    • Chrome (web)                       • chrome                    • web-javascript • Google Chrome 87.0.4280.141

• No issues found!
```

</details>

**Comment by darshankawar** [comment]
Verified this issue using code sample provided in original [description](https://github.com/flutter/flutter/issues/59137#issue-635969391) and running it on latest versions and I am unable to replicate the crash. Minimizing the app -> then opening it -> tapping on textformfield opens the keyboard properly without throwing any crash logs.

<details>
<summary> console log </summary>

```

Launching lib/main.dart on Darshan's iphone in debug mode...
Automatically signing iOS for device deployment using specified development team in Xcode project: X8NNQ9CYL2
Running Xcode build...
Xcode build done.                                           52.0s
Installing and launching...
(lldb) 2022-11-22 15:55:59.871533+0530 Runner[22394:2245917] Warning: Unable to create restoration in progress marker file
Debug service listening on ws://127.0.0.1:53428/kN0rsrJm404=/ws
Syncing files to device Darshan's iphone...
fopen failed for data file: errno = 2 (No such file or directory)
Errors found! Invalidating cache...
fopen failed for data file: errno = 2 (No such file or directory)
Errors found! Invalidating cache...
[VERBOSE-2:FlutterObservatoryPublisher.mm(138)] Could not register as server for FlutterObservatoryPublisher, permission denied. Check your 'Local Network' permissions for this app in the Privacy section of the system Settings.
[VERBOSE-2:FlutterObservatoryPublisher.mm(138)] Could not register as server for FlutterObservatoryPublisher, permission denied. Check your 'Local Network' permissions for this app in the Privacy section of the system Settings.
[Common] Snapshot request 0x283299560 complete with error: <NSError: 0x2832992f0; domain: FBSSceneSnapshotErrorDomain; code: 4; reason: "an unrelated condition or state was not satisfied">
[Common] Snapshot request 0x2832928b0 complete with error: <NSError: 0x283298750; domain: FBSSceneSnapshotErrorDomain; code: 4; reason: "an unrelated condition or state was not satisfied">
[Snapshotting] Snapshotting a view (0x11601be00, UIKeyboardImpl) that is not in a visible window requires afterScreenUpdates:YES.
[Snapshotting] Snapshotting a view (0x115866800, UIKeyboardImpl) that is not in a visible window requires afterScreenUpdates:YES.
Application finished.

```

</details>

<details>
<summary> stable, master flutter doctor -v </summary>

```
[✓] Flutter (Channel stable, 3.3.8, on macOS 12.2.1 21D62 darwin-x64, locale
    en-GB)
    • Flutter version 3.3.8 on channel stable at
      /Users/dhs/documents/fluttersdk/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 52b3dc25f6 (25 hours ago), 2022-11-09 12:09:26 +0800
    • Engine revision 857bd6b74c
    • Dart version 2.18.4
    • DevTools version 2.15.0

[!] Xcode - develop for iOS and macOS (Xcode 12.3)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    ! Flutter recommends a minimum Xcode version of 13.
      Download the latest version or update via the Mac App Store.
    • CocoaPods version 1.11.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] VS Code (version 1.62.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.21.0

[✓] Connected device (5 available)
    • SM G975F (mobile)       • RZ8M802WY0X • android-arm64   • Android 11 (API 30)
    • Darshan's iphone (mobile)  • 21150b119064aecc249dfcfe05e259197461ce23 •
      ios            • iOS 14.4.1 18D61
    • iPhone 12 Pro Max (mobile) • A5473606-0213-4FD8-BA16-553433949729     •
      ios            • com.apple.CoreSimulator.SimRuntime.iOS-14-3 (simulator)
    • macOS (desktop)            • macos                                    •
      darwin-x64     • Mac OS X 10.15.4 19E2269 darwin-x64
    • Chrome (web)               • chrome                                   •
      web-javascript • Google Chrome 98.0.4758.80

[✓] HTTP Host Availability
    • All required HTTP hosts are available

! Doctor found issues in 1 category.

[!] Flutter (Channel master, 3.6.0-7.0.pre.18, on macOS 12.2.1 21D62 darwin-x64,
    locale en-GB)
    • Flutter version 3.6.0-7.0.pre.18 on channel master at
      /Users/dhs/documents/fluttersdk/flutter
    ! Warning: `flutter` on your path resolves to
      /Users/dhs/Documents/Fluttersdk/flutter/bin/flutter, which is not inside
      your current Flutter SDK checkout at
      /Users/dhs/documents/fluttersdk/flutter. Consider adding
      /Users/dhs/documents/fluttersdk/flutter/bin to the front of your path.
    ! Warning: `dart` on your path resolves to
      /Users/dhs/Documents/Fluttersdk/flutter/bin/dart, which is not inside your
      current Flutter SDK checkout at /Users/dhs/documents/fluttersdk/flutter.
      Consider adding /Users/dhs/documents/fluttersdk/flutter/bin to the front
      of your path.
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 2921ca0c48 (2 hours ago), 2022-11-21 21:43:26 -0500
    • Engine revision 8dd8e092e4
    • Dart version 2.19.0 (build 2.19.0-415.0.dev)
    • DevTools version 2.19.0
    • If those were intentional, you can disregard the above warnings; however
      it is recommended to use "git" directly to perform update checks and
      upgrades.

[!] Xcode - develop for iOS and macOS (Xcode 12.3)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    ! Flutter recommends a minimum Xcode version of 13.
      Download the latest version or update via the Mac App Store.
    • CocoaPods version 1.11.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] VS Code (version 1.62.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.21.0

[✓] Connected device (5 available)
    • SM G975F (mobile)       • RZ8M802WY0X • android-arm64   • Android 11 (API 30)
    • Darshan's iphone (mobile)  • 21150b119064aecc249dfcfe05e259197461ce23 •
      ios            • iOS 14.4.1 18D61
    • iPhone 12 Pro Max (mobile) • A5473606-0213-4FD8-BA16-553433949729     •
      ios            • com.apple.CoreSimulator.SimRuntime.iOS-14-3 (simulator)
    • macOS (desktop)            • macos                                    •
      darwin-x64     • Mac OS X 10.15.4 19E2269 darwin-x64
    • Chrome (web)               • chrome                                   •
      web-javascript • Google Chrome 98.0.4758.80

[✓] HTTP Host Availability
    • All required HTTP hosts are available

! Doctor found issues in 1 category.



```

</details>

Closing this as fixed.
